Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/countdown/Countdown.tsx

```diff
@@ -24,20 +25,63 @@ interface ICountdownProps {
     handleOnCountdownChange?: (seconds: number) => void;
 }
 
+const remainingTimeClassName = 'remainingTime';
+
+const renderTimeOut = () => (
```

Comment:
This should be a useCallback function.

Reply:
After discussion we agreed this doesn't need to be a useCallback function.